### ISSUE TITLE
Hot Corner DnD wont work

### ISSUE THREAD

**Issue by alex285** [jderose9/dash-to-panel#84]
**Reproduce** 
1. Open Nautilus
2. DnD a file on left top corner 

**Outcome:**
Nothing happens

**Expected:**
Shell should go on overview mode and we should be able drop the file on some application

**Workaround:**
In Dash To Panel we should enable "Activities" and then we can DnD the file on left-bottom to succeed the action

**Ideas**
While it would be possible to add an invisible area on bottom left corner of the extension to fix this, could we restore the original functionality with the top left corner?

**Comment by alex285** [comment]
oh also to add, this happens on GNOME Shell 3.24 beta, that also has an API freeze

**Comment by gaveen** [comment]
I'm on Fedora 24 with GNOME Shell 3.20. I can confirm this issue is present there too.

**Comment by jderose9** [comment]
The top left hot corner drag-and-drop definitely doesn't work when the panel is in the bottom position. I looked into it far enough to see that gnome-shell doesn't call handleDragOver() on the hot corner when the panel isn't positioned underneath. I didn't dig any further than that because I thought making the entire panel open the overview in the same manner that the hot corner used to would be a better solution for most people (I did this work recently #46), and that most people using this extension probably have the hot corner disabled. I can leave this one open though if you disagree.
